### PR TITLE
Fix false/true error

### PR DESCRIPTION
**Pull request by alexvy86**
# Title
Fixed guidance for the ReadOnly property when implementing a custom read-only collection.

## Summary

Changed `false` to `true`.

### CODE REVIEW

**Review by jzeferino — APPROVED** [review]
LGTM.

File: docs/standard/design-guidelines/guidelines-for-collections.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Any type designed specifically to manipulate a group of objects having some comm
   
  **✓ DO** use <xref:System.Collections.ObjectModel.ReadOnlyCollection%601>, a subclass of `ReadOnlyCollection<T>`, or in rare cases `IEnumerable<T>` for properties or return values representing read-only collections.  
   
- In general, prefer `ReadOnlyCollection<T>`. If it does not meet some requirement (e.g., the collection must not implement `IList`), use a custom collection by implementing `IEnumerable<T>`, `ICollection<T>`, or `IList<T>`. If you do implement a custom read-only collection, implement `ICollection<T>.ReadOnly` to return false.  
+ In general, prefer `ReadOnlyCollection<T>`. If it does not meet some requirement (e.g., the collection must not implement `IList`), use a custom collection by implementing `IEnumerable<T>`, `ICollection<T>`, or `IList<T>`. If you do implement a custom read-only collection, implement `ICollection<T>.ReadOnly` to return true.
```
Comment (rpetrusha) [inline docs/standard/design-guidelines/guidelines-for-collections.md]: The name of the property is actually ICollection\<T\>.IsReadOnly.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thank you, @alexvy86, for contributing to the dotnet/docs repo and for correcting the incorrect Boolean value. The paragraph has another error, though: the name of the property is IsReadOnly, not ReadOnly. Would you mind changing that as well?

**Comment by rpetrusha** [comment]
Thanks, @alexvy86, for submitting your PR. I've made the additional change for you. Once an internal build completes successfully, we'll merge your PR.

**Comment by alexvy86** [comment]
Hey Ron, sorry about the delay, I couldn't go through all my emails in the last couple of days. Thanks for taking care of the additional change!

**Comment by mairaw** [comment]
Good to merge @rpetrusha?

**Comment by rpetrusha** [comment]
Yes, this is good to merge, @mairaw. And thanks for the contribution, @alexvy86. We'll merge your PR now. The change should appear on docs.microsoft.com in the next day or two.

**Review by rpetrusha — APPROVED** [review]

**Comment by alexvy86** [comment]
Happy to help!

**Pull request merged**